subprojects/common/core/src/main/java/hu/bme/mit/theta/core/type/fptype/FpRoundingMode.java

```diff
@@ -0,0 +1,13 @@
+package hu.bme.mit.theta.core.type.fptype;
+
+public enum FpRoundingMode {
+	RNE, /* Round nearest ties to even */
+	RNA, /* Round nearest ties to away */
+	RTP, /* Round toward positive */
+	RTN, /* Round toward negative */
+	RTZ; /* Round toward zero */
+
+	public static FpRoundingMode getDefaultRoundingMode() {
+		return RNE;
```

Comment:
This particular rounding mode is the default according to the C standard. Maybe it is worth mentioning somewhere in the docs or a comment here.
```suggestion
		return RNE; /* Default because it is default in the C standard */
```

Reply:
Fixed in [0d1af43a1]()